src/components/footer/Footer.js

```diff
@@ -8,6 +8,7 @@ export default function Footer() {
     <Fade bottom duration={1000} distance="5px">
     <div className="footer-div">
       <p className="footer-text">{emoji("Made with ❤️ by Saad Pasta")}</p>
+      <p className="footer-text"><a href="https://github.com/saadpasta/developerFolio">Created from saadpasta/developerFolio</a></p>
```

Comment:
Can we change this to something minimal? 
Like using a grey color font-awesome code icon
![image](https://user-images.githubusercontent.com/48270786/90306078-37035b80-dee7-11ea-84d8-2ee8aa2d9002.png)

Reply:
@kartikcho please see my recent change.  Suggesting a similar setting to https://beautifuljekyll.com

```
<p className="footer-text">Theme by <a href="https://github.com/saadpasta/developerFolio">developerFolio</a></p>
```